example_envs/tests/bookinfo_test.rs

```diff
@@ -5,13 +5,15 @@ use example_envs::productpage::ProductPage;
 use example_envs::reviews::Reviews;
 use queues::IsQueue;
 use rpc_lib::rpc::Rpc;
+use std::path::PathBuf;
 
 #[test]
 fn check_bookinfo() {
-    // Set up library access
+    // Set up plugin name
+    let plugin_str = "../target/debug/libfilter_example.dylib";
```

Comment:
I am surprised this passes. Is there some change in the way filters are loaded? dylibs only exist on MacOS.

Reply:
I just copy pasted from main - I fixed that now.  No, I didn't change anything from how filters are loaded;  the only thing I changed in the plugin file was adding routing info to RPCs to get the filter to accept them.

Reply:
Yeah we have some code that sets the extension based on the OS, so that is why it works. Was just surprised.